src/machine_charm.py

```diff
@@ -80,6 +80,18 @@ def on_remove(self, _) -> None:
         if self._is_installed:
             raise GrafanaAgentInstallError("Failed to uninstall grafana-agent")
 
+    def metrics_rules(self) -> list:
+        """Return a list of metrics rules."""
+        return []
```

Comment:
is this a `#TODO` or is this a method that has no meaning in the context of the machine charm, and if so, should it ever be called?

Reply:
It is a todo